### PR TITLE
Move and rename MT management

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR moves and renames the machine translation management section.

### Proposed changes

- Move the section to the settings area of menu bar.
- Rename it to Machine Translations.
- Remove the debug mode constraint.

### Side effects

- I hope none.

### Resolved issues

Fixes: #2118 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/cms/templates/_base.html [hunk 1]
```diff
@@ -211,12 +196,19 @@
                         {% translate "Imprint" %}
                     </a>
                 {% endif %}
-                {% if perms.cms.view_user or perms.cms.view_languagetreenode %}
+                {% if perms.cms.manage_translations or perms.cms.view_user or perms.cms.view_languagetreenode %}
                     <!-- Settings Section-->
```
Comment (JoeyStk) [inline integreat_cms/cms/templates/_base.html:200]: As @timoludwig suggest in the issue report I think we should also check if at least one MT provider has been enabled in this region. Therefore I think we should add the check "if DEEPL_ENABLED"

Comment (timoludwig) [reply integreat_cms/cms/templates/_base.html:200]: I'd show the entry when at least either DeepL or SUMM.AI are enabled.

File: integreat_cms/cms/templates/_base.html [hunk 2]
```diff
@@ -211,12 +196,19 @@
...
                     <div class="min-h-3 pl-2">
                         <p class="font-bold">{% translate "Settings" %}</p>
                     </div>
                 {% endif %}
+                {% if perms.cms.manage_translations %}
+                    <a href="{% url 'translations_management' region_slug=request.region.slug %}"
+                       class="{% if current_menu_item == 'translations_management' %} active{% endif %}">
+                        <i icon-name='bot'></i>
+                        {% translate "Machine Translations" %}
```
Comment (JoeyStk) [inline integreat_cms/cms/templates/_base.html:209]: I agree with @timoludwig in the issue report, that we should widen the navigation bar, so "machine translation" fits in one row

**Review by JoeyStk — CHANGES_REQUESTED** [review]

File: integreat_cms/cms/templates/_base.html (hunk 1 above)
Comment (JoeyStk) [reply integreat_cms/cms/templates/_base.html:200]: Oh yes, of course. I forgot about SUMM.AI for a second :see_no_evil:

Comment (MizukiTemma) [reply integreat_cms/cms/templates/_base.html:200]: Sorry, I overlooked it 🙈 
Now MT providers are checked too.

File: integreat_cms/cms/templates/_base.html (hunk 2 above)
Comment (MizukiTemma) [reply integreat_cms/cms/templates/_base.html:209]: Oh no, I missed this point too 😓 Now it's widened.

File: integreat_cms/cms/templates/_base.html [hunk 3]
```diff
@@ -211,12 +196,19 @@
                         {% translate "Imprint" %}
                     </a>
                 {% endif %}
-                {% if perms.cms.view_user or perms.cms.view_languagetreenode %}
+                {% if SUMM_AI_ENABLED and perms.cms.manage_translations or DEEPL_ENABLED and perms.cms.manage_translation or perms.cms.view_user or perms.cms.view_languagetreenode %}
```
Comment (timoludwig) [inline integreat_cms/cms/templates/_base.html:199]:
```suggestion
                {% get_mt_visibility request.region perms as show_mt_section %}
                {% if show_mt_section or perms.cms.view_user or perms.cms.view_languagetreenode %}
```
I suggest to also add a check whether summ ai is enabled in the current region and to move this complex condition into a template tag (e.g. to [base_filters.py](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/templatetags/base_filters.py)):
```
@register.simple_tag
def get_mt_visibility(region, perms):
    return "cms.manage_translations" in perms and (
        settings.DEEPL_ENABLED or settings.SUMM_AI_ENABLED and region.summ_ai_enabled
    )
```

File: integreat_cms/cms/templates/_base.html [hunk 4]
```diff
@@ -211,12 +196,19 @@
...
                     <!-- Settings Section-->
                     <div class="min-h-3 pl-2">
                         <p class="font-bold">{% translate "Settings" %}</p>
                     </div>
                 {% endif %}
+                {% if SUMM_AI_ENABLED and perms.cms.manage_translations or DEEPL_ENABLED and perms.cms.manage_translations %}
```
Comment (timoludwig) [inline integreat_cms/cms/templates/_base.html:205]:
```suggestion
                {% if show_mt_section %}
```

**Review by timoludwig — CHANGES_REQUESTED** [review]

File: integreat_cms/cms/templates/_base.html (hunk 3 above)
Comment (MizukiTemma) [reply integreat_cms/cms/templates/_base.html]: It's a good idea 😃 Thank you for suggestion!

File: integreat_cms/cms/templatetags/base_filters.py [hunk 5]
```diff
@@ -35,3 +36,22 @@ def get_private_member(element, key):
...
+    This tag checks whether either DEEPL or SUMM.AI is activated in the region and the user has permission to manage automatic translations.
+
+    :param region: The rcurrent region
+    :type region: ~integreat_cms.cms.models.regions.region.Region
+
+    :param perms: Permission of the user
+    :type perms: ~django.contrib.auth.context_processors.PermWrapper
+
+    :return: Whether the MT section should be shown
+    :rtype: boolean
```
Comment (timoludwig) [inline integreat_cms/cms/templatetags/base_filters.py:53]:
```suggestion
    :rtype: bool
```

File: integreat_cms/cms/templatetags/base_filters.py [hunk 6]
```diff
@@ -35,3 +36,22 @@ def get_private_member(element, key):
...
+@register.simple_tag
+def get_mt_visibility(region, perms):
+    """
+    This tag checks whether either DEEPL or SUMM.AI is activated in the region and the user has permission to manage automatic translations.
+
+    :param region: The rcurrent region
+    :type region: ~integreat_cms.cms.models.regions.region.Region
+
+    :param perms: Permission of the user
+    :type perms: ~django.contrib.auth.context_processors.PermWrapper
```
Comment (timoludwig) [inline integreat_cms/cms/templatetags/base_filters.py:50]: Could you add this to Sphinx' nitpicky ignore list?

**Review by timoludwig — CHANGES_REQUESTED** [review]

File: integreat_cms/cms/templatetags/base_filters.py (hunk 6 above)
Comment (MizukiTemma) [reply integreat_cms/cms/templatetags/base_filters.py:50]: Applied 👍 Thank you!

**Review by timoludwig — APPROVED** [review]

**Review by JoeyStk — CHANGES_REQUESTED** [review]
I noticed one more thing. Since the navigation bar is wider now, it gets very clear that the logo is centered and not aligned with the navigation links. After talking to @timoludwig I would suggest to remove bg-center and replace it with ml-2 [here](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/templates/_base.html#L104). 
grafik

**Comment by MizukiTemma** [comment]
@JoeyStk 

ohhhhn 🤦 now fixed!

**Review by JoeyStk — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**